src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -594,3 +594,50 @@ registerAction2(class RevealRunningCellAction extends NotebookAction {
 		}
 	}
 });
+
+registerAction2(class RevealLastFailedCellAction extends NotebookAction {
+	constructor() {
+		super({
+			id: REVEAL_RUNNING_CELL,
+			title: localize('revealLastFailedCell', "Go to Last Failed Cell"),
+			tooltip: localize('revealLastFailedCell', "Go to Last Failed Cell"),
+			shortTitle: localize('revealLastFailedCellShort', "See Last Failure"),
+			precondition: NOTEBOOK_LAST_CELL_FAILED,
+			menu: [
+				{
+					id: MenuId.EditorTitle,
+					when: ContextKeyExpr.and(
+						NOTEBOOK_IS_ACTIVE_EDITOR,
+						NOTEBOOK_LAST_CELL_FAILED,
+						ContextKeyExpr.notEquals('config.notebook.globalToolbar', true)
+					),
+					group: 'navigation',
+					order: 100
+				},
+				{
+					id: MenuId.NotebookToolbar,
+					when: ContextKeyExpr.and(
+						NOTEBOOK_IS_ACTIVE_EDITOR,
+						NOTEBOOK_LAST_CELL_FAILED,
+						ContextKeyExpr.equals('config.notebook.globalToolbar', true)
+					),
+					group: 'navigation/execute',
+					order: 100
+				},
+			],
+			icon: icons.warningIcon,
```

Comment:
also, is the colourless icon the best in this case? Or should it be something like a yellow colour?